src/js/components/Tag.js

```diff
@@ -0,0 +1,35 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+import pick from 'lodash/object/pick';
+import keys from 'lodash/object/keys';
+
+const CLASS_ROOT = "tag";
+
+export default class Tag extends Component {
+
+  render () {
+    var classes = [CLASS_ROOT];
+
+    if (this.props.className) {
+      classes.push(this.props.className);
+    }
+
+    var other = pick(this.props, keys(Anchor.propTypes));
+
+    return (
+      <div className={classes.join(' ')} onClick={this.props.onClick}>
+        {this.props.children}
+        <Anchor {...other} className="tag--label">
```

Comment:
can we replace `tag--label` by `${CLASS_ROOT}--label`, you can use string template in ES6 which is really cool for that. https://github.com/grommet/grommet/blob/master/src/js/components/Distribution.js#L381